### PR TITLE
fix GA runners

### PR DESCRIPTION
**Pull request by spacedragon**
# PR into Azure/azureml-examples

## Checklist

I have:

- [ ] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: cli/endpoints/online/model-1/environment/conda.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ dependencies:
   - scikit-learn=0.24.2
   - scipy=1.7.1
   - pip:
-    - azureml-defaults==1.33.0
+    - itsdangerous<2.0,>=0.24
```
Comment (rsethur) [inline cli/endpoints/online/model-1/environment/conda.yml:11]: Do we need the library itsdangerous? @spacedragon  @petrodeg

File: cli/endpoints/online/model-2/environment/conda.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ dependencies:
   - scikit-learn=0.24.2
   - scipy=1.7.1
   - pip:
-    - azureml-defaults==1.33.0
+    - itsdangerous<2.0,>=0.24
```
Comment (rsethur) [inline cli/endpoints/online/model-2/environment/conda.yml:11]: same as above: Do we need the library itsdangerous? @spacedragon @petrodeg

File: cli/endpoints/amlarc/model-1/environment/conda.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ dependencies:
   - scikit-learn=0.24.2
   - scipy=1.7.1
   - pip:
-    - azureml-defaults==1.33.0
+    - itsdangerous<2.0,>=0.24
```
Comment (rsethur) [inline cli/endpoints/amlarc/model-1/environment/conda.yml:11]: Do we need the library itsdangerous? @spacedragon @petrodeg

File: cli/endpoints/online/model-1/environment/conda.yml (hunk 1 above)
Comment (petrodeg) [reply cli/endpoints/online/model-1/environment/conda.yml:11]: Yes, we need to pin as it has broken all our runners. Klein has more details

**Pull request merged**